app/services/converters/order_detail_to_journal_rows.rb

```diff
@@ -0,0 +1,23 @@
+module Converters
+  class OrderDetailToJournalRows
+
+    attr_accessor :journal, :order_detail, :total
+
+    def initialize(journal, order_detail, options = {})
```

Comment:
I need to think a little more as I process how all this works, but my gut reaction is that this class shouldn't need the journal itself. I see we need it for the association, but I wonder if there's a way to do the association outside this class.